doc/langref.html.in

```diff
@@ -5980,12 +5979,12 @@ fn bang2() void {
       For the case when no errors are returned, the cost is a single memory write operation, only in the first non-failable function in the call graph that calls a failable function, i.e. when a function returning {#syntax#}void{#endsyntax#} calls a function returning {#syntax#}error{#endsyntax#}.
       This is to initialize this struct in the stack memory:
       </p>
-      {#syntax_block|zig|stack_trace_struct.zig#}
+      {#code_begin|syntax|stack_trace_struct.zig#}
 pub const StackTrace = struct {
     index: usize,
-    instruction_addresses: [N]usize,
+    instruction_addresses: []usize,
 };
-      {#end_syntax_block#}
+      {#code_end#}
       <p>
       Here, N is the maximum function call depth as determined by call graph analysis. Recursion is ignored and counts for 2.
```

Comment:
> This line now is referencing N which was removed

Thanks, I missed it.

Instead of changing the text to "The slice length is ...", it may be better to keep the current code. The problem is that the builtin type is not defined that way.

Reply:
It seems that the "Implementation Details" section is obsolete, since the generated function `__zig_return_error` was used only by stage1.